bin-src/lib/getStorybookMetadata.ts

```diff
@@ -170,12 +170,35 @@ export const getStorybookMetadata = async (ctx: Context) => {
   const r = typeof __non_webpack_require__ !== 'undefined' ? __non_webpack_require__ : require;
   const mainConfig = await r(path.resolve(configDir, 'main'));
 
-  const info = await Promise.all([
-    findAddons(ctx, mainConfig),
-    findConfigFlags(ctx),
-    findViewlayer(ctx),
-    findBuilder(mainConfig),
-  ]);
-  ctx.log.debug(info);
-  return info.reduce((acc, obj) => Object.assign(acc, obj), {});
+  let addons = {};
+  try {
+    addons = await findAddons(ctx, mainConfig);
+  } catch (e) {
```

Comment:
Would it be helpful to log out the stack trace too? Maybe we should add Sentry to CLI.

Reply:
adding stack trace now

re: sentry, I think that's a good idea -- I'd probably want to chat with Gert about where it makes sense to inject that into the context etc.